pkgs/applications/science/programming/scyther/cli.nix

```diff
@@ -0,0 +1,31 @@
+{ stdenv, lib, cmake, glibc, makeWrapper, fetchFromGitHub, flex, bison, source }:
+stdenv.mkDerivation { 
+  name = "scyther-cli-${source.rev}";
+  src = source;
+  preConfigure = ''
+    cd src
+    # Since we're not in a git dir, the normal command this project uses to create this file wouldn't work
+    printf "#define TAGVERSION \"${source.rev}\"\n" >> version.h
+
+    # For some reason it doesn't use the C89 standard which would allow __inline__
+    for file in *.*
+    do
+      substituteInPlace $file --replace "__inline__ " ""
```

Comment:
Please do an upstream PR for this, such that we don't need to patch it ourselves. The rest of your PR looks good enough.